packages/blocks/src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { PlusIcon, RepositoryIcon } from '@mergestat/icons';
+
+import { Button } from '../Button';
+
+type EmptyStateProps = {
+  onClick: () => void;
+};
+
+export const EmptyState: React.FC<
+  EmptyStateProps &
+    React.DetailedHTMLProps<
+      React.DetailsHTMLAttributes<HTMLElement>,
+      HTMLElement
+    >
+> = ({ onClick }) => {
+  return (
+    <div className="t-center">
+      <div className="t-circle-icon">
+        <RepositoryIcon className="t-icon" />
+      </div>
+      <p>No repositories added yet</p>
+      <Button
+        skin="secondary"
+        onClick={onClick}
+        startIcon={<PlusIcon className="t-icon" />}
+      >
+        Add Repository
+      </Button>
+    </div>
```

Comment:
The three "sections" should become props that can be passed in:

1. One for the "icon" - can be an arbitrary component, will most frequently be an icon
2. Text - to populate the `<p>...</p>`
3. Action - can be an arbitrary component, will most frequently be a button